compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -175,12 +173,43 @@ class ComposeScene internal constructor(
         }
     }
 
+    private val childScenes = mutableSetOf<ComposeScene>()
+
+    /**
+     * Adds a child [ComposeScene], so that nodes in it can be found in tests.
+     */
+    fun addChildScene(scene: ComposeScene) {
```

Comment:
This function doesn't have a strict defined behavior for external users. We may adding it because .

Solutions:
- make it `internal` for now.
- rename to addRootForTest (but I am very reluctant to do that, because we still need to think about how this function can be useful for external user)
- make a parallel tree of `roots` that doesn't live in `ComposeScene` (it is very vague idea, I don't know how it should look yet)

The easiest option is the first option.

Reply:
We also don't have public API for adding new owners directly to the ComposeScene, because the API wasn't well thought out. The only current way to add an owner - via popup.

There were a thought - adding a new owner via public function `scene.addRoot(content)`, which name is similar to `addChildScene`

Reply:
I think I'll make it `internal`.